home-manager/services/code-syncer/sync.sh

```
@@ -1,94 +1,336 @@
-CURSOR_USER_DIR="/Users/shunkakinoki/Library/Application Support/Cursor/User"
-WINDSURF_USER_DIR="/Users/shunkakinoki/Library/Application Support/Windsurf/User"
+VSCODE_USER_DIR="$HOME/Library/Application Support/Code/User"
+ANTIGRAVITY_USER_DIR="$HOME/Library/Application Support/Antigravity/User"
+CURSOR_USER_DIR="$HOME/Library/Application Support/Cursor/User"
+WINDSURF_USER_DIR="$HOME/Library/Application Support/Windsurf/User"
 
-# Files to sync
+# Files
 SETTINGS_FILE="settings.json"
 KEYBINDINGS_FILE="keybindings.json"
 EXTENSIONS_FILE="extensions.list"
```

Shellcheck (0.11.0)

[warning] EXTENSIONS_FILE appears unused. Verify use (or export if used externally).

(SC2034)